scripts/micro_benchmarks.py

```diff
@@ -221,14 +222,15 @@ def build(
         verbose: bool) -> None:
     '''Restores and builds the benchmarks'''
 
-    packages = path.join(get_repo_root_path(), 'packages')
+    packages = path.join(get_artifacts_directory(), 'packages')
 
     if incremental == 'no':
         __log_script_header("Removing packages, bin and obj folders.")
         binary_folders = [
             packages,
             path.join(BENCHMARKS_CSPROJ.working_directory, 'bin'),
```

Comment:
After these changes, do we need this line: `path.join(BENCHMARKS_CSPROJ.working_directory, 'bin'),`?

Reply:
probably we can delete it.